app/controllers/signup/memberships_controller.rb

```diff
@@ -3,18 +3,14 @@ class MembershipsController < AbstractSignupController
     content_security_policy false
 
     def new
-      @membership = FakeMembership.new
```

Comment:
This is just cleaning up some cruft that I hacked together fast for the August demo. It needs more cleanup, but this will do for now.